plugins/social_network/tag_visitor.cpp

```diff
@@ -327,12 +329,21 @@ namespace golos { namespace plugins { namespace social_network { namespace tags
         update_tags(op.author, op.permlink);
     }
 
-    comment_metadata get_metadata(const comment_api_object &c) {
+    comment_metadata get_metadata(const comment_api_object &c, database& db_) {
         comment_metadata meta;
 
-        if (!c.json_metadata.empty()) {
+        const auto& idx = db_.get_index<comment_content_index>().indices().get<by_id>();
+
+        auto itr = idx.find(c.id);
+        if (itr == idx.end()) {
+            // throw exception?
+        }
+
+        comment_content_object content = *itr;
+
+        if (!content.json_metadata.empty()) {
             try {
-                meta = fc::json::from_string(c.json_metadata).as<comment_metadata>();
+                meta = fc::json::from_string(to_string(content.json_metadata)).as<comment_metadata>();
             } catch (const fc::exception& e) {
                 // Do nothing on malformed json_metadata
```

Comment:
I got a question here why do we do nothing on malformed json_metadata? How do we even proceed since comment_metadata object is not properly initialized? This question depends on my other question about should i throw an exception if comment content object is not found in database or what